proposals/lip-0051.md

```diff
@@ -2625,7 +2625,19 @@ def beforeCrossChainMessageForwarding(trs: Transaction, ccm: CCM, ccmFailed: boo
 
 ### Endpoints for Off-Chain Services
 
-This section specifies the non-trivial or recommended endpoints of the Token module and does not include all endpoints. TBA
+This section specifies the non-trivial or recommended endpoints of the Token module and does not include all endpoints.
+
+#### getBalances
+
+This function returns available and locked balances for all tokens associated with a given address.
```

Comment:
This essentially outputs all entries of the user substore where the key contains the `address`, I would treat is as trivial.

Reply:
OK, will remove it. I agree it is trivial...